pkg/sso/client.go

```diff
@@ -227,3 +231,104 @@ func (c *Client) PromoteDraftConnection(ctx context.Context, opts PromoteDraftCo
 
 	return workos.TryGetHTTPError(res)
 }
+
+// CreateConnectionOpts contains the options to activate a Draft Connection.
+type CreateConnectionOpts struct {
+	Source string `json:"source"`
+}
+
+// ConnectionDomain represents the domain records associated with a Connection.
+type ConnectionDomain struct {
+	// Connection Domain unique identifier.
+	ID string `json:"id"`
+
+	// Domain for a Connection record.
+	Domain string `json:"domain"`
+}
+
+// ConnectionStatus represents a Connection's linked status.
+type ConnectionStatus string
+
+// Constants that enumerate the available Connection's linked statuses.
+const (
+	Linked   ConnectionStatus = "linked"
+	Unlinked ConnectionStatus = "unlinked"
+)
+
+// Connection represents a Connection record.
+type Connection struct {
+	// Connection unique identifier.
+	ID string `json:"id"`
+
+	// Connection linked status.
+	Status ConnectionStatus `json:"status"`
+
+	// Connection name.
+	Name string `json:"name"`
+
+	// Connection provider type.
+	ConnectionType ConnectionType `json:"connection_type"`
+
+	// OAuth Client ID.
+	OAuthUID string `json:"oauth_uid"`
+
+	// OAuth Client Secret.
+	OAuthSecret string `json:"oauth_secret"`
+
+	// OAuth Client Redirect URI.
+	OAuthRedirectURI string `json:"oauth_redirect_uri"`
+
+	// Identity Provider Issuer.
+	SamlEntityID string `json:"saml_entity_id"`
+
+	// Identity Provider SSO URL.
+	SamlIdpURL string `json:"saml_idp_url"`
```

Comment:
```suggestion
	SamlIDPURL string `json:"saml_idp_url"`
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
Ah good catch. Updated.